includes/WikiSEO.php

```diff
@@ -154,6 +156,8 @@ public function setMetadata( array $metadataArray, ParserOutput $out = null ): v
 	public function addMetadataToPage( OutputPage $out ): void {
 		$this->modifyPageTitle( $out );
 
+		$this->loadDescriptionFromApi( $out->getTitle() );
```

Comment:
`addMetadataToPage` runs only when the parserfunction is present.
In the current state this would not set a description without `{{#seo:}}` :/

Reply:
`addMetadataToPage` runs from the `onBeforePageDisplay` hook. I tested and it works without the tag.

Reply:
Ah, you are right, forgot the hook!